lua/astrocommunity/pack/astro/README.md

```diff
@@ -0,0 +1,8 @@
+# Astro Language Pack
+
+This plugin pack does the following:
+
+- Adds `astro` Treesitter parser
+- Adds `astro` language server
+- Adds `js-debug-adapter` for debugging
+- Adds [nvim-dap-vscode-js](https://github.com/mxsdev/nvim-dap-vscode-js) for debugging
```

Comment:
Should this line be there? It doesn't seem like this pack adds this plugin

Reply:
No it shouldn't be, fixed.